HoloPipelines/pipelineList.json

```diff
@@ -0,0 +1,38 @@
+{
+   "p0":

```

Comment:
Can I ask you for another favour? Could you please use expressive ids? In fact, please just take the fields you use for name, like `nifti2obj`, and then use human-readable names for the `name` field, like "NIfTI to DICOM Conversion". Please remember that this will also mean a few other files need to be changed, for instance the tests.

Reply:
All IDs renamed to the values from the 'name' field. And name has been updated with a more readable value.